docs/changenodehostname.rst

```diff
@@ -0,0 +1,112 @@
+Change the hostname of a cluster node
+=====================================
+
+On the node
+^^^^^^^^^^^
+
+Change the hostname:
+
+.. code-block:: shell
+
+        $ hostnamectl set-hostname <your hostname, here: ControlPlane2>
```

Comment:
A code block's body should be indented by 3 characters only (so the contents line up with the `code-block` directive)